timesketch/frontend-ng/src/components/Explore/SearchDropdown.vue

```diff
@@ -33,6 +33,26 @@ limitations under the License.
       </v-col>
       <v-divider vertical></v-divider>
 
+      <v-col v-if="matches.timeFilters.length" cols="4">
```

Comment:
The columns look like they wrap around now that a 4th is added:
<img width="1677" height="990" alt="image" src="https://github.com/user-attachments/assets/f2120256-1836-4a31-ba1d-1be086e8f367" />

Reply:
fixed